afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+gravid	2
+lykkeligere	2
+ny sesong	1
+overveldet	2
+støtten	1
+støtte	2
+godta	2
+kolegevinst	1
+kunnskapsløftet	1
+kommer til norge	1
+kreftfri	3
+prisvinner	1
+gullvinner	2
+stafettgull	2
+spurtslå	3
+rekord	2
+fantomløp	3
+rørende	2
+utrolig	3
+sier ja til	4
+overveldende	4
+banebrytende	3
+klart flertall	2
+pirrende	1
+terningkast fire	2
+terningkast fem	3
+terningkast seks	4
+fremgangsrik	3
+fremgangsrike	3
+best	3
+godt	3
+færre sykemeldinger	2
+utrettelige	2
+vakre	2
+intelligente	2
+hylles	2
+rørende	2
+velsignet	2
+tar til tårene	2
+gruser	1
+viktigste	2
+endelig	2
+pionéren	2
+pionér	2
+bot og bedring	2
+stjerne	1
+barske	1
+stråler	3
+avslørt	2
+avsløre	2
+invitert	1
+sjekk resultatet	1
+trosser	2
+trosse	2
+her er resultatet	1
+gjenforent	1
+varmerekord	1
+knuste alle	2
+hedersprisen	2
+hederspris	2
+varmebølgen	1
+tilbake for fullt	3
+drømmesommer	2
+motvirke	2
+spanderer	1
+fosser frem	1
+slå ring om	1
+viktige	2
+rekordmange	2
+enormt	1
+engasjement	1
+blomsterstøtten	1
+blomsterstøtte	1
+vinterolympier	1
+utklasset	2
+taktiske	1
+taktisk	1
+genistrek	2
+rekordhøy	1
+ekstase	1
+knalltall	1
+gullfangsten	2
+le	1
+ler	1
+banditt	-3
+krigen	-2
+mister	-2
+dør	-2
+slaktes	-3
+harnisk	-3
+totalforbud	-2
+frykten	-3
+frykte	-2
+strengere	-2
+strenge	-1
+skutt	-3
+stokkeslag	-2
+ulykken	-3
+bekjempe	-1
+opportunisme	-2
+overvurdert	-2
+skutt	-2
+lidelsen	-2
+skadd	-2
+båtulykke	-2
+voldsomt	-1
+svindlet	-3
+unødvendige	-2
+bekymringer	-2
+ødelegg	-2
+mister retten	-2
+får stryk	-2
+terningkast en	-4
+terningkast to	-3
+terningkast tre	-2
+masete	-2
+maset	-2
+spyttklyse	-3
+hører ikke hjemme	-2
+skjules	-2
+sykdommene	-2
+forbys	-1
+under press	-2
+falske	-2
+svindelforsøk	-2
+bortføres	-3
+drepes	-3
+oppetter ryggen	-3
+kaldere samfunn	-2
+bekymringsmelding	-2
+steine	-2
+bli full	-2
+søpla	-1
+bot	-2
+skjerp deg	-2
+drittlei	-2
+sexmeldinger	-2
+sexmelding	-2
+hetere	-2
+steinkasterne	-2
+ikke invitert	-3
+aldri opplevd maken	-3
+utskjelt	-2
+raser	-2
+rasende	-2
+fattige	-2
+fattig	-1
+sølvskje	-1
+rikeste	-1
+tatt	-1
+knuste	-1
+søppelmaten	-2
+søppelmat	-2
+ingenting	-1
+kastes ut	-2
+konfronteres	-1
+konfrontere	-1
+motsagt	-1
+mistillitsforslaget	-2
+mistillitsforslag	-2
+mistillit	-2
+gå av	-2
+unødvendig	-2
+gir blaffen	-2
+gått for langt	-2
+gikk for langt	-2
+høyrepopulismens	-2
+tråkk på	-2
+trakke ned	-2
+ikke tillit	-3
+avgang	-1
+sur	-1
+sure	-2
+junkfood	-2
+knebling	-2
+kneblet	-2
+motbør	-1
+kutte	-1
+ødelegges	-2
+forbud	-2
+forbudet	-1
+kritiske	-2
+fikk ikke	-1
+hundebæsjen	-1
+hundebæsj	-1
+bæsjen	-1
+bæsj	-1
+ransforsøk	-2
+legger ned	-2
+gebyr	-1
+døden	-3
+ingen fordeler	-3
+svinedyrt	-2
+svindyrt	-2
+ekstraskatt	-2
+uforståelig	-2
+skodrama	-1
+derfor skal ikke	-1
+verste	-2
+byggesprekk	-1
+depresjon	-2
+virusinfeksjon	-2
+arrogansen	-2
+ulvestrid	-2
+pesten	-3
+sky dem som	-1
+panikkangst	-2
+varslene	-1
+har fått nok	-1
+sverte	-2
+svertekampanje	-2
+fortviler	-2
+avvikling	-1
+ryktene	-1
+hardporno	-2
```

Comment:
Should this be negative?

Reply:
Removed for now. As I deal mainly with news, if this is mentioned it is usually in a negative context.